modules/040-control-plane-manager/hooks/effective_kubernetes_version.go

```diff
@@ -73,6 +72,10 @@ const minK8sVersionRequirementKey = "controlPlaneManager:minUsedControlPlaneKube
 const maxUsedK8sVersionSecretKey = "maxUsedControlPlaneKubernetesVersion"
 const deckhouseDefaultK8sVersionSecretKey = "deckhouseDefaultKubernetesVersion"
 
+// This value is set on the controller build in the deckhouse-controller/go-build.sh script.
+// Do not touch it !!!
+var DefaultKubernetesVersion = "1.30"
```

Comment:
hm, let it move to global variables in deckhouse controller, not only in hooks?
In https://github.com/deckhouse/deckhouse/blob/77ecc874a4e1a331b6c510deeeff616a400176a2/deckhouse-controller/pkg/controller/controller.go#L72-L71

Or we will have cyclic dependency error?